### ISSUE TITLE
Error in Taxonomic network visualization workflow

### ISSUE THREAD

**Issue by emankhalaf** [microbiome/microbiome#87]
Hello,

I try to generate network graph based on taxonomy (phylum /genus level). I am following the steps in this link:
https://microbiome.github.io/microbiome/Networks.html
however, I stuck from the beginning when creating igraph
## Create graph object
n <- net$refit
colnames(n) <- rownames(n) <- rownames(otu)

I got this error:
Error in `rownames<-`(`*tmp*`, value = c("7ff649b304795c8bfef9c98fe2365355", : attempt to set 'rownames' on an object with no dimensions.

this long code is the feature ID (otu ID).

I wonder how could I fix this error.
Although the package is really helpful, but I think there is always something missing in the code??

Your swift response is much appreciated.
Best
Eman

**Comment by antagomir** [comment]
I manage to run this with the example data by following the tutorial step-by-step. Are you using the same data? If not, can you compare to the example data set to see what is the difference? If yes, then we have to investigate further why it does not work in all systems.

**Comment by emankhalaf** [comment]
I am using my data not the demo. I realized that also in DesSeq workflow of microbiome package, when run it, it did not work. But when run Deseq with phyloseq workflow,  it worked and I found many steps are missed in the microbiome/microbiome workflow.
The package introduces excellent functions for microbiome analysis, but unfortunately not all of them are feasible.
I hope it comes to a completion very soon.
Eman
